website/docs/d/nodes.html.markdown

```diff
@@ -1,31 +1,31 @@
 ---
 layout: "consul"
-page_title: "Consul: consul_catalog_nodes"
-sidebar_current: "docs-consul-data-source-catalog-nodes"
+page_title: "Consul: consul_nodes"
+sidebar_current: "docs-consul-data-source-nodes"
```

Comment:
You also need to update these in website/consul.erb or the sidebar highlighting won't work anymore.

Reply:
Fixed, thanks!